web/src/index.js

```diff
@@ -0,0 +1,80 @@
+import { trace, context } from '@opentelemetry/api';
+import { tracer } from './tracing-http';
+import { determineMessage } from './message';
+import { determineYear } from './year';
+import { determineName } from './name';
+
+const parentSpan = tracer.startSpan('main');
+const ctx = trace.setSpan(context.active(), parentSpan);
+const documentLoadSpan = tracer.startSpan('document_load', undefined, ctx);

```

Comment:
So when using the web SDK, you can actually replace this all with `startActiveSpan`! Might be worth trying out